test/index.spec.js

```diff
@@ -63,20 +63,6 @@ describe('multihashing', () => {
   })
 
   describe('invalid arguments', () => {
-    it('returns an error on non implemented func', (done) => {
-      multihashing(new Buffer('beep boop'), 'blake2b', (err) => {
```

Comment:
Instead of removing this test case, can we try to use a function that we still don't have?